src/main/java/pl/agh/edu/iisg/io/vmms/vmmsbackend/controller/EmailController.java

```diff
@@ -0,0 +1,100 @@
+package pl.agh.edu.iisg.io.vmms.vmmsbackend.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import com.sendgrid.*;
+import pl.agh.edu.iisg.io.vmms.vmmsbackend.exception.InvalidEmailRequestException;
+import pl.agh.edu.iisg.io.vmms.vmmsbackend.exception.MailSendingFailureException;
+import pl.agh.edu.iisg.io.vmms.vmmsbackend.model.MailSubject;
+import pl.agh.edu.iisg.io.vmms.vmmsbackend.model.VMAdmin;
+import pl.agh.edu.iisg.io.vmms.vmmsbackend.dto.EmailConfigurationDTO;
+import pl.agh.edu.iisg.io.vmms.vmmsbackend.service.MailSubjectService;
+import pl.agh.edu.iisg.io.vmms.vmmsbackend.service.VMAdminService;
+
+import java.io.IOException;
+import java.util.List;
+
+@CrossOrigin("*")
+@RestController
+@RequestMapping("/email")
+public class EmailController {
+    private final Email from;
+    private final SendGrid sg;
+
+    private final MailSubjectService mailSubjectService;
+    private final VMAdminService vmAdminService;
+
+    @Autowired
+    EmailController(MailSubjectService mailSubjectService, VMAdminService vmAdminService){
+        this.mailSubjectService = mailSubjectService;
+        this.vmAdminService = vmAdminService;
+
+        from = new Email("complaint@vmms.ki.agh.edu.pl");
+        String apiKey;
+        try {
+            apiKey = System.getenv("SENDGRID_API_KEY");
+            if (apiKey.equals("")) {
+                throw new NullPointerException();
+            }
+        } catch (NullPointerException e){
+            throw new RuntimeException("Failed to access SendGrid API key! " +
+                    "Please set SENDGRID_API_KEY environment variable");
+        }
+        sg = new SendGrid(apiKey);
+    }
+
+    @RequestMapping(method = RequestMethod.POST)
+    @ResponseStatus(HttpStatus.CREATED)
+    public void sendMail(@RequestParam("subject") String subjectKey,
```

Comment:
I believe that it would be better to extract this to an Email service that would allow us to easily send emails from the app. It could be useful for some errors logging etc.